quest/util/config.py

```diff
@@ -160,13 +159,13 @@ def save_settings(filename=None):
 
 def _default_config_file():
     base = get_settings()['BASE_DIR']
-    return os.path.join(base, 'quest_config.yml')
+    return os.path.join(base, './settings/quest_config.yml')
```

Comment:
Avoid using the slashes when constructing a path. That is the point of `os.path.join`. Try doing this instead: `os.path.join(base, 'settings', 'quest_config.yml')`